src/main/java/org/jocl/Sizeof.java

```diff
@@ -28,330 +28,345 @@
 package org.jocl;
 
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 /**
  * Size constants for scalar and vector data types.
  */
 public final class Sizeof 
 {
+
+    /**
+     * The logger used in this class
+     */
+    private static final Logger logger =
+            Logger.getLogger(Sizeof.class.getName());
+
     static
     {
         CL.loadNativeLibrary();
     }
     
     /**
-     * Size of a cl_char, in bytes.

```

Comment:
This will certainly not be changed. There are several "violations" of the standard Java naming conventions (most obviously: All the `cl_...` classes). But these violations have been intentional, to mimic the original API. One could argue that this "was a bad idea", but I don't think so: Following the original naming makes unambiguously clear that a `cl_mem` is just a `cl_mem`, and not an object-oriented `CLMemoryObject` with additional functionality. 

But regardless of that: Changing these constants would **severely** break the API. (I'd rather consider *adding* the new ones, like in `java.awt.Color.red` and `java.awt.Color.RED`. But I think this is not necessary here either).


(I've just started looking over the changes, and this one jumped into my eye. I will have a closer look at the other ones in the next days)